src/layouts/partials/downloads.html

```diff
@@ -1,8 +1,108 @@
 <section class="download-section" id="downloads">
-    <div class="container">
+    <div class="my-5">
 
-        <h1>DCRDEX is available in Decrediton or as a standalone application</h1>
-        <h2>v0.6 is now released!</h2>
+        {{ $bwlogo := resources.Get "/images/bw-logo.svg" }}
+        <img src="{{ $bwlogo.Permalink }}" class="mt-4">
```

Comment:
This needs some css for mobile.

Please add `id="bisonlogo"` to the img tag and the following somewhere in `downloads.scss`:

```css
#bisonlogo {
    width: 100%;
    max-width: 441px;
}
```